e2e/ctl_v3_auth_test.go

```diff
@@ -739,3 +740,36 @@ func authTestWatch(cx ctlCtx) {
 	}
 
 }
+
+func authTestLease(cx ctlCtx) {
```

Comment:
this should probably be in `integration/v3_auth_test.go` using raw grpc calls since it's testing etcdserver behavior, not etcdctl